packages/app/src/lobby/lobbyPage.tsx

```diff
@@ -35,9 +35,9 @@ const LobbyPage: React.FunctionComponent = () => {
 
 	for (let i = players.length; i < MAX_PLAYERS_IN_GAME; i++) {
 		botElements.push(
-			<div key={i} className="player bot">
-				<span className="name">empty slot</span>
-			</div>
+			<LobbyPlayerBanner
```

Comment:
This also needs a key! Even if you're not using `map`, you need a key if you're rendering things inside an array (which we are, as `botElements` is an array)